src/test/java/edu/wright/airviewer2/SplitAndMergeTest.java

```diff
@@ -0,0 +1,123 @@
+package edu.wright.airviewer2;
+
+import static org.junit.Assert.*;
+
+
+import java.io.ByteArrayOutputStream;
+import java.io.File;
+import java.io.IOException;
+import java.io.OutputStream;
+import java.nio.file.Path;
+import java.nio.file.Paths;
+import java.text.SimpleDateFormat;
+import java.util.ArrayList;
+import java.util.Date;
+import java.util.Iterator;
+import java.util.List;
+
+import org.apache.pdfbox.io.MemoryUsageSetting;
+import org.apache.pdfbox.multipdf.PDFMergerUtility;
+import org.apache.pdfbox.multipdf.Splitter;
+import org.apache.pdfbox.pdmodel.PDDocument;
+import org.junit.Test;
+import org.testfx.framework.junit.ApplicationTest;
+
+import javafx.application.Platform;
+import javafx.scene.Group;
+import javafx.scene.control.Alert;
+import javafx.scene.control.Pagination;
+import javafx.scene.control.Alert.AlertType;
+import javafx.scene.image.ImageView;
+import javafx.stage.DirectoryChooser;
+import javafx.stage.FileChooser;
+
+
+
+public class SplitAndMergeTest extends ApplicationTest {
+	
+	
+	@Test
+	public void mergefileTest()  {
+		try {
+		Pagination pagination=  new Pagination();
+		Path currentDir = Paths.get(".");
+		String current = currentDir.toAbsolutePath() + "/src/test/resources/test.pdf";
+		Path exist = Paths.get(current);
+		AIRViewerModel model = new AIRViewerModel(exist);
+		pagination.setPageCount(model.numPages());
+	    pagination.setDisable(true);
+	    Group pageImageGroup = new Group();
+	    pagination.setPageFactory(index -> {
+	    	System.out.println("haiinpagena");
+            ImageView currentPageImageView = new ImageView(model.getImage(0));
+            pageImageGroup.getChildren().clear();
+            pageImageGroup.getChildren().add(currentPageImageView);
+   
```

Comment:
code lines are not structured. plrase align

Reply:
Formatted the code and committed the changes